infra/email_receipt_inbox/lambdas/parsers/parse_restaurant_platforms.py

```
@@ -0,0 +1,447 @@
+    return out
+
+
+# ---------------------------------------------------------------- SCE (utility)
+def parse_sce(msg, lines, subject):
+    out = {"platform": "SCE", "merchant_name": "Southern California Edison",
+           "items": [], "subtotal": None, "tax": None, "tip": None,
+           "order_id": None, "grand_total": None}
+    joined = " ".join(lines)
+    low = subject.lower()
+
+    if "payment" in low or any("Amount Paid" in l for l in lines):
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)